lms/urls.py

```diff
@@ -10,7 +10,8 @@
 if settings.DEBUG or settings.FEATURES.get('ENABLE_DJANGO_ADMIN_SITE'):
     admin.autodiscover()
 
-urlpatterns = ('',  # nopep8
+urlpatterns = (
+    '',  # nopep8
```

Comment:
Yes indeed - pylint it is - with the first argument in its original location:
```
83: C0330: (bad-continuation), : Wrong continued indentation.
85: C0330: (bad-continuation), : Wrong continued indentation.
```

Where line 83 is the newly added line, and
line 85 is the closing paren

I'll revert and put in the disable for that warning.